README.md

```diff
@@ -9,13 +16,23 @@
 * Duplicate the .env.example: `cp .env.example .env`
 * For running sail commands, by default, you are required to enter the full path to the executable in `vendor/bin/sail`. Most devs prefer to create an alias in their shell so they only have to type `sail`. Read the sail docs about [configuring a sail alias](https://laravel.com/docs/11.x/sail#configuring-a-shell-alias). Further documentation will assume an alias exists in your shell.
 
-## Updating the Project
-* If you get an error with the `composer.lock` file, you may need to update your project
-* To update, navigate to the project directory and run `composer update`
+## Installing Current Project Dependencies
+* All project contributors should run these commands every week to ensure your local project is using the current project dependencies.
+  * `composer install`
+  * `npm install`
+  * `npm run build`
```

Comment:
Zach was hitting an error with this that he posted in Good Dads.  It might be good to include that in a Troubleshooting section.

Reply:
I ran a git reset --hard and reinstalled all packages. It worked, all clear,

Reply:
Thank you for the update, Zach!